README.md

```diff
@@ -189,37 +186,61 @@ otherwise.
+```js
+//What should each of the following expressions return? Once you have an answer, copy the expression in question into the console and hit enter to see if you're right!
+
+// Practice with AND
+0 && false;
+
+0 && true;
+
+true && NaN;
+
+true && !1;
+
+!0 && "This is a string";
+
+!0 && "";
+
+!0 && !!"";
+
+// Practice with OR
+0 || false;
+
+true || false;
+
+true || 1;
+
+!true || !false
+
+!1 || !0
+```
 
 Before we get to that, however...
 
 ### A Note About Comments in Code
 
-Note that most of the lines in the REPL currently have the characters `//` at
-the front. The `//` is used in JavaScript to 'comment out' lines of code. Any
+Note that some of the lines in the code above currently have the characters `//`
+at the front. The `//` is used in JavaScript to 'comment out' lines of code. Any
 line in your code that has `//` at the front will _not be interpreted_ by the
 JavaScript engine. This enables us to do two things: 1) include text (i.e.,
-messages meant for humans) in our code files without JavaScript throwing an
-error, and 2) _temporarily_ keep lines of code from being interpreted; this
-second use can be especially handy when debugging code.
-
-Try removing the `//` at the beginning of line 1 and clicking the Run button.
-Not surprisingly, JavaScript doesn't know what to do with that line of 'code'.
-The `//` enables programmers to include messages to other programmers — or to
-their own future selves — for documentation or clarification purposes. Don't
-forget to re-comment out line 1 before you move on.
+messages meant for humans like we have above) in our code files without
+JavaScript throwing an error, and 2) _temporarily_ keep lines of code from being
+interpreted; this second use can be especially handy when debugging code. You'll
+learn more about this later.
 
 ### Instructions
 
-Okay, let's get some practice with logical operators. Take a look at line 4 in
-the REPL (the only un-commented out line). What do you think the expression will
-return? Think it through and come up with an answer, then click Run to check
-whether you're right. When you're done with the first expression, work your way
-down, un-commenting out each expression in turn. Be sure you think each example
-through and figure out your answer before clicking Run.
+Okay, let's get some practice with logical operators. Take a look at the 4th
+line in the code above (the only un-commented out line). What do you think the
+expression will return? Think it through and come up with an answer, then copy &
+paste it into your console and hit enter to check whether you're right. When
+you're done with the first expression, work your way down, un-commenting out
+each expression in turn. Be sure you think each example through and figure out
```

Comment:
We need to remove the "un-commenting out..." part. Maybe "work your way down through the remaining expressions" instead.

Reply:
```suggestion
expression will return? Think it through and come up with an answer, then copy &
paste it into your console and hit enter to check whether you're right. When
you're done with the first expression, work your way down through the remaining 
expression in turn. Be sure you think each example through and figure out
```